guides/common/modules/con_remote-execution-workflow.adoc

```diff
@@ -1,6 +1,9 @@
 [id="remote-execution-workflow_{context}"]
 = Remote Execution Workflow
 
+For custom Ansible roles that you create, or roles that you download, you must install the package containing the roles on your {SmartProxyServer}.
```

Comment:
You can also place Ansible roles using `scp` or `wget` onto your SmartProxyServer. I am not sure if mentioning "packages" is really necessary. 

I like that you mention "you have to place them onto your SmartProxyServer" before you can/have to import them.

Reply:
Let's track this in https://github.com/theforeman/foreman-documentation/issues/2766